### ISSUE TITLE
PF2e: Santize GM Rolls does not santize enough

### ISSUE THREAD

**Issue by themartyr781** [sPOiDar/fvtt-module-hide-gm-rolls#38]
With Sanitize GM Rolls disabled the players will see
Melee Strike: Kukri
Target: Player (AC)
Result: Hit/miss by number
Traits: Attack, Agile, Finesse, etc.
1d20+modifier
total result rolled.

With Sanitize GM Rolls enabled the player will see
Melee Strike: Kukri
Target: Player (AC)
Result: Hit/miss by number
total result rolled.

result: Hit/miss by number and total result rolled is information that the player should not have. It informs them in a way that allows them to perform things like Raising a Shield because they know the difference between a hit and a miss.

When Sanitizing, I'd prefer to see something more in line with the Flat Check module (https://foundryvtt.com/packages/pf2-flat-check). Just a simple colored box that indicates in words Critical Miss, Critical Hit, Miss, or Hit with no math shown. In this way Sanitizing actually mimics an in person GM roll behind the screen.

Thank you for considering this request.

**Comment by sPOiDar** [comment]
Yeah PF2E adds a lot of custom extra elements to their roll cards, and last time I looked there wasn't a good way to distinguish between the different kinds of data they output there, making it complicated/brittle to hide the right elements. I know another user of this module was going to talk to the PF2E devs about adding markup to their output to make it possible to distinguish between different element types, but I'm not sure that went anywhere.

I'll have another look, because it's been a while, but if things are in the same state as they used to be, this will be a problematic request.
